pkg/lib/cockpit-po-plugin.js

```diff
@@ -25,8 +25,10 @@ module.exports = class {
                 },
                 () => {
                     const linguas_file = path.resolve(srcdir, "po/LINGUAS");
+                    compilation.fileDependencies.add(linguas_file);
                     const linguas = fs.readFileSync(linguas_file, 'utf8').match(/\S+/g);
                     const po_files = linguas.map(lang => path.resolve(srcdir, 'po', lang + '.po'));
+                    po_files.forEach(po_file => compilation.fileDependencies.add(po_file));
```

Comment:
It seems equally simple and effective to just add all *.po files as dependencies.

Reply:
The reason for this is that a glob result doesn't have a timestamp: we can't teach make to check if the glob result changed.  `LINGUAS` has exactly this purpose, and we should use it.

We could potentially add `LINGUAS` to the `Makefile` dependencies and still use the glob, but it wouldn't bring as much: it would just move the crash from `webpack-make` to the `Makefile` telling us that it doesn't know how to get the `LINGUAS` file.

Reply:
It's worth mentioning: I came across this as an actual bug: I pulled the cockpit repo after a .po refresh and was surprised to find that nothing got rebuilt.

Reply:
But LINGUAS only gets updated rarely, when a new language gets added. It does not change on po file updates. So how does its timestamp help us in any way?